quiz.js

```diff
@@ -0,0 +1,50 @@
+
+
+var treeDiv = document.getElementById("treeDiv");
+
+var lineMaker = function(limitz, stringToAdd) {
+	var stringMade = ''
+	for (var i = 0; i < limitz; i++) {
+		stringMade += stringToAdd;
+	};
+	return stringMade;
+};
+
+var makeTree = function () {
```

Comment:
**Requirement not met:** Create a tree function that should build a pine tree out of a character in the Chrome dev tools console. It accepts a single object as an argument. The object should have two key/value pairs.